[2021-12-28 14:44:30,750] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_cavan.scrap_data manual__2021-12-28T14:38:19.542256+00:00 [queued]>
[2021-12-28 14:44:30,765] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_cavan.scrap_data manual__2021-12-28T14:38:19.542256+00:00 [queued]>
[2021-12-28 14:44:30,766] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 14:44:30,767] {taskinstance.py:1242} INFO - Starting attempt 2 of 4
[2021-12-28 14:44:30,768] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 14:44:30,784] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-28 14:38:19.542256+00:00
[2021-12-28 14:44:30,794] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_cavan', 'scrap_data', 'manual__2021-12-28T14:38:19.542256+00:00', '--job-id', '5135', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_cavan.py', '--cfg-path', '/tmp/tmp4z95w2mx', '--error-file', '/tmp/tmps8xegukp']
[2021-12-28 14:44:30,796] {standard_task_runner.py:77} INFO - Job 5135: Subtask scrap_data
[2021-12-28 14:44:30,790] {standard_task_runner.py:52} INFO - Started process 60372 to run task
[2021-12-28 14:44:30,897] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_cavan.scrap_data manual__2021-12-28T14:38:19.542256+00:00 [running]> on host 0c125a5bba95
[2021-12-28 14:44:31,064] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_cavan
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-28T14:38:19.542256+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-28T14:38:19.542256+00:00
[2021-12-28 14:44:31,085] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 14:44:31,087] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py cavan
[2021-12-28 14:44:32,684] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 14:44:32,845] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 14:44:32,846] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 14:44:32,847] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 14:44:32,848] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 14:44:32,849] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 14:44:32,934] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 14:44:32,935] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 14:44:32,936] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 14:44:32,937] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 14:44:32,938] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 14:44:32,939] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 14:44:32,940] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 14:44:32,941] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 14:44:32,942] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 14:44:32,943] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 14:44:32,944] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 14:44:32,945] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 14:44:32,946] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 14:44:32,947] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 14:44:32,947] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 14:44:32,948] {spark_submit.py:514} INFO - files                   null
[2021-12-28 14:44:32,949] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 14:44:32,950] {spark_submit.py:514} INFO - archives                null
[2021-12-28 14:44:32,951] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 14:44:32,951] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 14:44:32,952] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-28 14:44:32,953] {spark_submit.py:514} INFO - childArgs               [cavan]
[2021-12-28 14:44:32,957] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 14:44:32,958] {spark_submit.py:514} INFO - packages                null
[2021-12-28 14:44:32,959] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 14:44:32,960] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 14:44:32,961] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 14:44:32,962] {spark_submit.py:514} INFO - 
[2021-12-28 14:44:32,962] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 14:44:32,963] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 14:44:32,964] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 14:44:32,965] {spark_submit.py:514} INFO - 
[2021-12-28 14:44:32,967] {spark_submit.py:514} INFO - 
[2021-12-28 14:44:33,310] {spark_submit.py:514} INFO - Main class:
[2021-12-28 14:44:33,312] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 14:44:33,313] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 14:44:33,315] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 14:44:33,316] {spark_submit.py:514} INFO - null
[2021-12-28 14:44:33,317] {spark_submit.py:514} INFO - cavan
[2021-12-28 14:44:33,318] {spark_submit.py:514} INFO - --verbose
[2021-12-28 14:44:33,319] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 14:44:33,320] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-28 14:44:33,321] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 14:44:33,322] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 14:44:33,322] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 14:44:33,323] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 14:44:33,325] {spark_submit.py:514} INFO - 
[2021-12-28 14:44:33,326] {spark_submit.py:514} INFO - 
[2021-12-28 14:44:33,327] {spark_submit.py:514} INFO - 
[2021-12-28 14:44:34,576] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 14:44:34,594] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 14:44:34,710] {spark_submit.py:514} INFO - 21/12/28 14:44:34 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 14:44:34,897] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO ResourceUtils: ==============================================================
[2021-12-28 14:44:34,898] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 14:44:34,899] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO ResourceUtils: ==============================================================
[2021-12-28 14:44:34,899] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-28 14:44:34,931] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 14:44:34,945] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 14:44:34,946] {spark_submit.py:514} INFO - 21/12/28 14:44:34 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 14:44:35,031] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 14:44:35,032] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 14:44:35,033] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 14:44:35,034] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 14:44:35,035] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 14:44:35,423] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO Utils: Successfully started service 'sparkDriver' on port 40953.
[2021-12-28 14:44:35,470] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 14:44:35,526] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 14:44:35,562] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 14:44:35,563] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 14:44:35,579] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 14:44:35,617] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-7001629c-cc94-4747-a14e-1f91636cf196
[2021-12-28 14:44:35,646] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 14:44:35,668] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 14:44:35,946] {spark_submit.py:514} INFO - 21/12/28 14:44:35 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-28 14:44:35,961] {spark_submit.py:514} INFO - 21/12/28 14:44:35 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2021-12-28 14:44:36,049] {spark_submit.py:514} INFO - 21/12/28 14:44:36 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4041
[2021-12-28 14:44:36,355] {spark_submit.py:514} INFO - 21/12/28 14:44:36 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 14:44:36,475] {spark_submit.py:514} INFO - 21/12/28 14:44:36 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 14:44:36,476] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 14:44:36,477] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 14:44:36,479] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 14:44:36,480] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 14:44:36,481] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 14:44:36,481] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 14:44:36,483] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 14:44:36,484] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 14:44:36,485] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 14:44:36,486] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 14:44:36,487] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 14:44:36,487] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 14:44:36,488] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 14:44:36,489] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 14:44:36,490] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 14:44:36,491] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 14:44:36,492] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 14:44:36,492] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 14:44:36,493] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 14:44:36,494] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 14:44:36,495] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 14:44:36,495] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 14:44:36,496] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 14:44:36,497] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 14:44:36,498] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 14:44:36,498] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 14:44:36,499] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 14:44:36,500] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 14:44:36,500] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 14:44:36,501] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 14:44:36,502] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 14:44:36,503] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 14:44:36,503] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 14:44:36,504] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 14:44:36,504] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 14:44:36,505] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 14:44:36,506] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 14:44:36,507] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 14:44:36,508] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 14:44:36,508] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 14:44:36,509] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 14:44:36,510] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 14:44:36,511] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 14:44:36,511] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 14:44:36,512] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 14:44:36,513] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 14:44:36,513] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 14:44:36,514] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 14:44:36,514] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 14:44:36,515] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 14:44:36,516] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 14:44:36,517] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 14:44:36,517] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 14:44:36,518] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 14:44:36,519] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 14:44:36,520] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 14:44:36,520] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 14:44:56,354] {spark_submit.py:514} INFO - 21/12/28 14:44:56 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 14:44:56,367] {spark_submit.py:514} INFO - 21/12/28 14:44:56 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 14:44:56,368] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 14:44:56,369] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 14:44:56,370] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 14:44:56,371] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 14:44:56,371] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 14:44:56,372] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 14:44:56,373] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 14:44:56,374] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 14:44:56,375] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 14:44:56,375] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 14:44:56,376] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 14:44:56,376] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 14:44:56,377] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 14:44:56,378] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 14:44:56,378] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 14:44:56,379] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 14:44:56,379] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 14:44:56,380] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 14:44:56,381] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 14:44:56,381] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 14:44:56,382] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 14:44:56,383] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 14:44:56,383] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 14:44:56,384] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 14:44:56,384] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 14:44:56,385] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 14:44:56,386] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 14:44:56,386] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 14:44:56,387] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 14:44:56,387] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 14:44:56,388] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 14:44:56,389] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 14:44:56,389] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 14:44:56,390] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 14:44:56,391] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 14:44:56,391] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 14:44:56,392] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 14:44:56,392] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 14:44:56,393] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 14:44:56,394] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 14:44:56,395] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 14:44:56,395] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 14:44:56,396] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 14:44:56,397] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 14:44:56,397] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 14:44:56,398] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 14:44:56,398] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 14:44:56,399] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 14:44:56,400] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 14:44:56,400] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 14:44:56,401] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 14:44:56,401] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 14:44:56,402] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 14:44:56,403] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 14:44:56,403] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 14:44:56,404] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 14:44:56,405] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 14:45:16,322] {spark_submit.py:514} INFO - 21/12/28 14:45:16 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 14:45:16,338] {spark_submit.py:514} INFO - 21/12/28 14:45:16 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 14:45:16,340] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 14:45:16,341] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 14:45:16,342] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 14:45:16,342] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 14:45:16,343] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 14:45:16,344] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 14:45:16,345] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 14:45:16,346] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 14:45:16,347] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 14:45:16,348] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 14:45:16,349] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 14:45:16,350] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 14:45:16,351] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 14:45:16,352] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 14:45:16,353] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 14:45:16,354] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 14:45:16,354] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 14:45:16,355] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 14:45:16,356] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 14:45:16,357] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 14:45:16,358] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 14:45:16,358] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 14:45:16,359] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 14:45:16,360] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 14:45:16,361] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 14:45:16,361] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 14:45:16,362] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 14:45:16,363] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 14:45:16,364] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 14:45:16,364] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 14:45:16,365] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 14:45:16,366] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 14:45:16,366] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 14:45:16,367] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 14:45:16,368] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 14:45:16,368] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 14:45:16,369] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 14:45:16,370] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 14:45:16,370] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 14:45:16,371] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 14:45:16,372] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 14:45:16,372] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 14:45:16,373] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 14:45:16,374] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 14:45:16,375] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 14:45:16,375] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 14:45:16,376] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 14:45:16,377] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 14:45:16,377] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 14:45:16,378] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 14:45:16,379] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 14:45:16,379] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 14:45:16,380] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 14:45:16,381] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 14:45:16,381] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 14:45:16,382] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 14:45:16,383] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 14:45:36,298] {spark_submit.py:514} INFO - 21/12/28 14:45:36 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 14:45:36,300] {spark_submit.py:514} INFO - 21/12/28 14:45:36 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 14:45:36,312] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 44435.
[2021-12-28 14:45:36,313] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO NettyBlockTransferService: Server created on 0c125a5bba95:44435
[2021-12-28 14:45:36,315] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 14:45:36,325] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4041
[2021-12-28 14:45:36,332] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 44435, None)
[2021-12-28 14:45:36,336] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 14:45:36,340] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:44435 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 44435, None)
[2021-12-28 14:45:36,343] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 14:45:36,345] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 44435, None)
[2021-12-28 14:45:36,347] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 44435, None)
[2021-12-28 14:45:36,351] {spark_submit.py:514} INFO - 21/12/28 14:45:36 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 14:45:36,371] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 14:45:36,394] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO MemoryStore: MemoryStore cleared
[2021-12-28 14:45:36,395] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManager: BlockManager stopped
[2021-12-28 14:45:36,406] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 14:45:36,411] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 14:45:36,428] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 14:45:36,562] {spark_submit.py:514} INFO - 21/12/28 14:45:36 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 14:45:36,563] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 14:45:36,565] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 14:45:36,566] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 14:45:36,567] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 14:45:36,568] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 14:45:36,569] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 14:45:36,570] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 14:45:36,571] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 14:45:36,572] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 14:45:36,573] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 14:45:36,574] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 14:45:36,575] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 14:45:36,576] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 14:45:36,577] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 14:45:36,578] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 14:45:36,579] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 14:45:36,580] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 14:45:36,581] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO SparkContext: SparkContext already stopped.
[2021-12-28 14:45:36,582] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 14:45:36,583] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-28 14:45:36,584] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-28 14:45:36,585] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 14:45:36,586] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 14:45:36,587] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 14:45:36,588] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 14:45:36,589] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 14:45:36,590] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 14:45:36,591] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 14:45:36,592] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 14:45:36,593] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 14:45:36,593] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 14:45:36,594] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 14:45:36,595] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 14:45:36,595] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 14:45:36,596] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 14:45:36,597] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 14:45:36,598] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 14:45:36,598] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 14:45:36,599] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 14:45:36,600] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 14:45:36,600] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 14:45:36,601] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 14:45:36,602] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 14:45:36,603] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 14:45:36,603] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 14:45:36,604] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 14:45:36,605] {spark_submit.py:514} INFO - 
[2021-12-28 14:45:36,659] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 14:45:36,661] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO ShutdownHookManager: Deleting directory /tmp/spark-09ec6379-c885-4f44-9035-2c405fb0fb2c
[2021-12-28 14:45:36,666] {spark_submit.py:514} INFO - 21/12/28 14:45:36 INFO ShutdownHookManager: Deleting directory /tmp/spark-abac0c50-b9d7-48b4-af99-cdb13ad17979
[2021-12-28 14:45:36,783] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py cavan. Error code is: 1.
[2021-12-28 14:45:36,791] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_cavan, task_id=scrap_data, execution_date=20211228T143819, start_date=20211228T144430, end_date=20211228T144536
[2021-12-28 14:45:36,813] {standard_task_runner.py:91} ERROR - Failed to execute job 5135 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py cavan. Error code is: 1.
[2021-12-28 14:45:36,866] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 14:45:36,977] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
